pkg/agent/variables.go

```diff
@@ -62,7 +62,6 @@ func getCustomDataVariables(config *datamodel.NodeBootstrappingConfiguration) pa
 	}
 
 	if !cs.Properties.IsVHDDistroForAllNodes() {
```

Comment:
I don't think we really need this conditional at all anymore...

Reply:
at this point the agent pools included in `Properties` should only include the single one for which agentbaker is serving the bootstrapping request

Reply:
following up offline, this block and all referenced files can potentially be cleaned up, but I'd do that in a separate pr.